erica/application/JobService/job.py

```diff
@@ -45,7 +45,9 @@ async def perform_job(request_id: UUID, repository: base_repository_interface, s
                 exc_info=True
             )
             entity.error_code = e.generate_error_response().get('message')
-            entity.error_message = e.generate_error_response().get('validation_problems', e.generate_error_response().get('message'))
+            entity.error_message = e.generate_error_response().get('message')
```

Comment:
I know this was already there before, but why do we call `e.generate_error_response()` several times? Couldn't we just do something like `error_response = e.generate_error_response()` and then extract message and validation problems from there?

Reply:
Good remark. 👍